.github/custom-issue-template/frontend-kit/3. Add a greeting component page.md

```diff
@@ -0,0 +1,16 @@
+# Background
```

Comment:
this and the new two routing tasks are very small (after you do one, the other are so similar and easy). Splitting them in 3 will actually take longer than one person to do it all together (just my thoughts)